### PR TITLE
Add stakes

### LINKED ISSUE #29: Add stakes table

### PR DESCRIPTION
**Pull request by shanev**
Fixes #29 

TODO
- [x] test with live chain
- [x] update stakebird dependecy after its merged in

NOTE: `update_at` isn't updated during an upsert. According to the SQL Boiler docs, it should. Anyone know why?

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -23,3 +23,5 @@ require (
 )
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.2-alpha.regen.4
+
+replace github.com/public-awesome/stakebird => ../stakebird
```
Comment (shanev) [inline go.mod]: Will update this after the chain is merged in and before merging this in.

**Review by JakeHartnell — APPROVED** [review]
LGTM! :christmas_tree:

File: workqueue/workqueue.go [hunk 2]
```diff
@@ -267,6 +268,52 @@ func handleUpvote(ctx context.Context, db *sql.DB, attributes []sdk.Attribute, h
...
+	model := &models.Stake{
+		Height:    height,
+		VendorID:  vendorID,
+		PostID:    postID,
+		Delegator: delegator,
+		Validator: validator,
+		Amount:    amount,
+	}
+
+	return model.Upsert(ctx, db, true, []string{"vendor_id", "post_id"}, boil.Whitelist("amount"), boil.Infer())
```
Comment (jhernandezb) [inline workqueue/workqueue.go]: Regarding your question about updated_at, you will need to add it to the whitelist `boil.Whitelist("amount", "updated_at")`

**Review by jhernandezb — APPROVED** [review]
Looks good!

**Pull request merged**